services/grafana-loki/metadata.yaml

```diff
@@ -2,6 +2,8 @@ displayName: Grafana Loki
 description: Horizontally-scalable, highly-available, multi-tenant log aggregation system inspired by Prometheus. Requires Logging Operator.
 category:
   - logging
+requiredDependencies:
+  - rook-ceph-cluster
```

Comment:
its possible to deploy Loki without rook-ceph, but it requires custom configuration to point Loki at another storage backend. will `requiredDependencies` allow that?

Reply:
No it will not. I can update. But we might want to update docs as well. cc @gvasquezvargas